GNP1/donations.yml

```diff
@@ -97,3 +97,157 @@
      - GNP1/OCD_UK.png   
   announcement_url: https://grahamsnumberplus1.com/charity-donations/
   charity_tx_id: "#28734"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Charlotte/Zara Marathon and OCD UK
+    address: Harvest Barn Chevin Green Farm Chevin Road Belper Derbyshire DE56 2UN
+    tel: +44 800 0487012
+    contact: no email provided
+  ada: 49.82
+  usd: 54.30
+  timestamp: 1619201249 
+  files:
+     - GNP1/Forwebpage.png   
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "192945107"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Mind in Harrogate
+    address: The Acorn Centre, 101A Station Parade, Harrogate, HG1 1HB
+    tel: +44 1423 503335
+    contact: tina@mindinharrogate.org.uk
+  ada: 87.61
+  usd: 108.64
+  timestamp: 1620828329 
+  files:
+     - GNP1/Mind_in_Harrogate.png  
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "IDN035332117000"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Bipolar UK
+    address: 32 Cubitt Street London WC1X0LR
+    tel: none supplied
+    contact: info@bipolaruk.org
+  ada: 53.90
+  usd: 73.30
+  timestamp: 1619880809 
+  files:
+     - GNP1/Bipolar.png
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "Non Supplied"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Strongminds
+    address: 515 Valley Street Suite 200 Maplewood NJ 07040 USA
+    tel: +1 862-222-4906
+    contact: Rasa@strongminds.org
+  ada: 51.90
+  usd: 86.15
+  timestamp: 1625141669 
+  files:
+     - GNP1/Strongminds.png
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "#1722-3011"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Mental Health Foundation
+    address: Colechurch House, 1 London Bridge Walk, London, SE1 2SX
+    tel: +44 20 7803 1100
+    contact:  events@mentalhealth.org.uk
+  ada: 51.03
+  usd: 91.86
+  timestamp: 1620814349 
+  files:
+     - GNP1/Mental_Health_Foundation.png
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "None Supplied"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Beating Eating Disorders
+    address: Unit 1 Chalk Hill House 19 Rosary Road Norwich Norfolk NR1 1SZ
+    tel: +44 1603 753308
+    contact:  fundraising@beateatingdisorders.org.uk
+  ada: 61.18
+  usd: 106.45
+  timestamp: 1622629109 
+  files:
+     - GNP1/Beat_Eating_Disorder.png
+  announcement_url: https://grahamsnumberplus1.com/charity-donations/
+  charity_tx_id: "Non Supplied"
+  
+- ticker: GNP1 
+  recipient: 
+    name: Anxiety UK
+    address: Nunes House 447 Chester Road Manchester M16 9HA
+    tel: +44 161 2267727
+    contact:  fundraising@beateatingdisorders.org.uk
+  ada: 52.07
+  usd: 82.28
+  timestamp: 1623711600 
+  files:
+     - GNP1/Anxiety_UK.png
```

Comment:
Got to say that in comparison to the other files this one doesn't really give that much...